### ISSUE TITLE
DANK tipping in profile/[FEATURE]

### ISSUE THREAD

**Issue by Bradymck** [district0x/memefactory#559]
# Problem

After we have user profiles, having an easy way to tip DANK to other users would be nice to have. 
## Implementation
This could be integrated in a similar way that 'Send to a friend' is integrated. Or other platforms that let you easily tip in app. 

Known steps towards feature implementation.
We need user profiles complete before we can do this. 

## Acceptance Criteria

Rules for the future PR to be accepted.

## Notes

Random notes to keep in mind while implementing it. Mostly about related issues and future plans and thoughts.

## Future Steps

Steps which should be taken after this issue has been resolved. Is it linked to other issue?
